src/Entity/Comment/Comment.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace App\Entity\Comment;
+
+class Comment
+{
+    //Attributes
+    protected $userId;
+    protected $postId;
+    protected $content;
+    protected $publishAt;
+    protected $status;
+
+    /**
+     * @param $userId
+     * @param $postId
+     * @param $content
+     * @param $publishAt
+     * @param $status
+     */
+    public function __construct($userId=NULL, $postId=NULL, $content=NULL, $publishAt=NULL, $status=NULL)
```

Comment:
Can you add some types ? if there are no types , then a can put an objet, float, string, ... , it must be accepted.